core/templates/domain/exploration/written-translation.model.spec.ts

```diff
@@ -13,33 +13,31 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for the WrittenTranslation object factory.
+ * @fileoverview Unit tests for the written-translation.model.
  */
 
 import {
-  WrittenTranslationObjectFactory,
   WrittenTranslation,
   TRANSLATION_DATA_FORMAT_HTML,
   TRANSLATION_DATA_FORMAT_UNICODE,
   TRANSLATION_DATA_FORMAT_SET_OF_NORMALIZED_STRING,
   TRANSLATION_DATA_FORMAT_SET_OF_UNICODE_STRING
-} from 'domain/exploration/WrittenTranslationObjectFactory';
+} from 'domain/exploration/written-translation.model';
 
 describe('WrittenTranslation object factory', () => {
-  let wtof: WrittenTranslationObjectFactory;
+  let writtenTranslation1: WrittenTranslation;
   let writtenTranslation: WrittenTranslation;
```

Comment:
Can we provide more specific names (instead of appending a number) e.g. describing what the writtenTranslation represents?

Reply:
@kevintab95 Changed writtenTranslation to writtenTranslationDict as it is a dict